test/src/sender/main_sender.cpp

```diff
@@ -22,10 +22,14 @@
 #include <cadmium/logger/tuple_to_ostream.hpp>
 #include <cadmium/logger/common_loggers.hpp>
 
-#include "../../../lib/vendor/NDTime.hpp"
+#include "../../../lib/DESTimes/include/NDTime.hpp"
 #include "../../../lib/vendor/iestream.hpp"
 #include "../../../include/data_structures/message.hpp"
 #include "../../../include/atomics/sender_cadmium.hpp"
+#include "../../../include/output_modification/convert_output.hpp"
+
+#define SIMULATOR_OUTPUT_PATH "./test/data/sender_test_output.txt"
```

Comment:
name of #define is not self-explanatory. This is sender component and #define statement must be consistent with it.

eg SENDER_OUTPUT_FILE_PATH

make changes in all files.

Reply:
use SIMULATOR_OUTPUT_PATH instead of hardcoding in line 82

static std::ofstream out_data("test/data/sender_test_output.txt");